binding.gyp

```diff
@@ -14,7 +33,7 @@
         "./build/yara/include"
       ],
       "libraries": [
-        "-lmagic",
+        "../build/libmagic.a",
```

Comment:
Do not link dynamically against `libmagic`, but rather use the provided `a` file for static binding. It is copied a bit earlier in the process - see the `action_before_build` pseudo-target.